bin/express

```diff
@@ -98,10 +99,10 @@ function createApplication(app_name, path) {
   }
 
   // JavaScript
-  var app = loadTemplate('js/app.js');
-  var www = loadTemplate('js/www');
-  var index = loadTemplate('js/routes/index.js');
-  var users = loadTemplate('js/routes/users.js');
+  var app = program.es6 ? loadTemplate('es6/app.js') : loadTemplate('js/app.js');
+  var www = program.es6 ? loadTemplate('es6/www') : loadTemplate('js/www');
+  var index = program.es6 ? loadTemplate('es6/routes/index.js') : loadTemplate('js/routes/index.js');
+  var users = program.es6 ? loadTemplate('es6/routes/users.js') : loadTemplate('js/routes/users.js');
```

Comment:
If everything is going to be loaded from a split directory, is it possible to make this more maintainable? Perhaps just decide the base template directory first and then load the files from within that instead of having a ternary on every line?